test/e2e/adminapi_cluster_getlogs.go

```diff
@@ -0,0 +1,118 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"net/http"
+	"net/url"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+var _ = Describe("[Admin API] Kubernetes get pod logs action", func() {
+	BeforeEach(skipIfNotInDevelopmentEnv)
+
+	const containerName = "e2e-test-container-name"
+	const podName = "e2e-test-pod-name"
+
+	When("in a standard openshift namespace", func() {
+		const namespace = "default"
+
+		It("should be able to get logs from a container of a pod", func() {
+			testGetPodLogsOK(containerName, podName, namespace)
+		})
+	})
+
+	When("in a customer namespace", func() {
+		const namespace = "e2e-test-namespace"
+
+		It("should be not be able to get logs from customer workload namespaces", func() {
+			testGetPodLogsFromCustomerNamespaceForbidden(containerName, podName, namespace)
+		})
+	})
+})
+
+// We will create a pod with known logs of its container and will compare the logs gotten through the kubernetes client and through the Admin API.
+func testGetPodLogsOK(containerName, podName, namespace string) {
+	By("creating a pod in default namespace with some known logs then comparing those logs with the logs of the container in the pod created")
+	pod := mockPod(containerName, podName, namespace)
+	pod, err := clients.Kubernetes.CoreV1().Pods(namespace).Create(context.Background(), pod, metav1.CreateOptions{})
```

Comment:
I think each call doesn't deserve own context. I would suggest to create it once somewhere in `It` and pass into the functions such as `testGetPodLogsOK`, `testGetPodLogsFromCustomerNamespaceForbidden`, etc as a param.

Reply:
+1

Reply:
ohh yeah .. my bad.. made the changes..